components/TemplateGrid/TemplateDesigner/TemplateGridGhostItem.vue

```diff
@@ -100,30 +85,28 @@ export default {
 
             return column + 1 === position.column;
         },
-        addGhostElementIfNeeded() {
-            if (typeof this.draggedElement === 'object') {
-                const elementsGap = 16;
-                const {
-                    width, height,
-                } = this.$el.getBoundingClientRect();
-                const {
-                    width: draggedElementWidth, height: draggedElementHeight,
-                } = this.draggedElement;
-                const normalizedWidth = width * draggedElementWidth - elementsGap;
-                const normalizedHeight = height * draggedElementHeight - elementsGap;
+        addGhostElement() {
+            let draggedElementWidth = 1;
+            let draggedElementHeight = 1;
 
-                addGhostElementToDraggableLayer({
-                    top: this.$el.offsetTop + (elementsGap / 2),
-                    left: this.$el.offsetLeft + (elementsGap / 2),
-                    width: normalizedWidth,
-                    height: normalizedHeight,
-                });
-            }
-        },
-        removeGhostElementIfExist() {
             if (typeof this.draggedElement === 'object') {
-                removeGhostElementFromDraggableLayer();
+                draggedElementWidth = this.draggedElement.width;
+                draggedElementHeight = this.draggedElement.height;
             }
+
+            const elementsGap = 16;
```

Comment:
Maybe better is set this variable as data state. You can use this as globally settings, and in the future move this to settings file.

Reply:
I would rather to keep it in method scope